sandbox.rmd

```diff
@@ -227,3 +236,59 @@ simbindone(x, bmax, kd, 0.01, 5, log = T)
 
 ```
 
+Maybe we just need to keep it to  method = auto
+
+This works, but clearly this is imperfect
+
+```{r}
+simbindone <- function(x, bmax, kd, cv, reps, log=F) {
+  
+  y = bmax*x/(kd + x)
+  
+  values <- data.frame(x=rep(x, reps), y)
+  
+  values <- dplyr::mutate(values, y=apply(values, 1, function(x) stats::rnorm(1, x[2], cv*x[2])))
+  
+  for (i in 1) {
+    if (log) {
+      model <- stats::lm(log(y) ~ log(x), data=values)
```

Comment:
We log x only, to flip the x-axis so the dose response is S shaped rather than hyperbolic. We don't log values of y.

Reply:
This fix is only to generate values (stored in model) that are then used to be the start values for the nls call in the next line. The final graph does not have log y values